src/test/resources/features/comp/CCH.feature

```diff
@@ -3,22 +3,35 @@ Feature: CCH
 
   Background:
     Given I am logged into "CS" as user "COMP_USER"
-    And I create a "COMP" case and move it to the "CCH" stage
+    When I create a "COMP" case and move it to the "CCH (From Service Triage)" stage
```

Comment:
not sure why we have switched the And and When around in this feature?
The 'When' should be the trigger for the expected scenario result, which I think was more accurate when the action step was the 'When' than the create case step

Reply:
Changed